src/hmem.c

```diff
@@ -528,12 +528,74 @@ bool ofi_hmem_is_initialized(enum fi_hmem_iface iface)
 	return hmem_ops[iface].initialized;
 }
 
+#include <stdio.h>
+
+void ofi_hmem_set_iface_filter(const char* iface_filter_str, bool* filter)
+{
+	int iface, rlen, llen;
+	char* entry = NULL;
+	const char* token = ";";
+	const char* iface_labels[ARRAY_SIZE(hmem_ops)] = {
+		"system",   // FI_HMEM_SYSTEM
+		"cuda",     // FI_HMEM_CUDA
+		"rocr",     // FI_HMEM_ROCR
+		"ze",       // FI_HMEM_ZE
+		"neuron",   // FI_HMEM_NEURON
+		"synapseai" // FI_HMEM_SYNAPSEAI
+	};
+
+	/* initialize array */
```

Comment:
All the comments in the commit are unnecessary.

Reply:
That's fine. I lean toward more comments than less. I'll remove them.